README.md

```diff
@@ -25,12 +25,15 @@
 ```jsonc
 "swipeToNavigate.action": "disabled" | "tabs" |  "grouped-tabs" | "recent-code" | "recent-files"
 ```
-
+```jsonc
+"swipeToNavigate.directionReverse": true | false
+```
 ### Examples:
 
 * Currently opened *tabs* - **default**.
 ```jsonc
 "swipeToNavigate.action": "tabs"
+"swipeToNavigate.directionReverse": false
```

Comment:
From a config point of view, it can be hard to understand what exactly you are reversing. Would you be interested in  entertaining the notion of calling this `previousSwipeLeftToRight`.

Just to clarify: I'm also not comfortable with `previousSwipeLeftToRight`. But I dislike ambiguity especially if its context based. 

Either this, or rephrase the docs with something else.